src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/DataSetVersionServiceTests.cs

```diff
@@ -0,0 +1,110 @@
+#nullable enable
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Public.Data;
+using GovUk.Education.ExploreEducationStatistics.Admin.Tests.Fixture;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests.Fixtures;
+using Microsoft.Extensions.DependencyInjection;
+using static GovUk.Education.ExploreEducationStatistics.Common.Utils.EnumUtil;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services;
+
+public class DataSetVersionServiceTests(TestApplicationFactory testApp) : IntegrationTestFixture(testApp)
+{
+    public class GetDataSetImportsInProgressTests(TestApplicationFactory testApp) : DataSetVersionServiceTests(testApp)
+    {
+        /// <summary>
+        /// Test that each "terminal" import status (i.e. statuses whereby the importing process has finished
+        /// processing for some reason) reports that no imports are in progress. 
+        /// </summary>
+        [Fact]
+        public async Task GetStatusesForReleaseVersion()
+        {
+            await GetEnums<DataSetVersionStatus>()
+                .ToAsyncEnumerable()
+                .ForEachAwaitAsync(AssertDataSetVersionStatusReturnedOk);
+        }
+        
+        /// <summary>
+        /// Test the scenario when data set imports are in progress, but for an unrelated ReleaseVersion.
+        /// </summary>
+        [Fact]
+        public async Task DataSetVersionForDifferentReleaseVersion()
```

Comment:
I think the name of this test class and the doc comments need updating here as they all still refer to import statuses.

I think ideally this test class wouldn't be needed if there were controller integration tests covering the release checklist. Is that what you were thinking too?

Reply:
Ahh right you are!  I've updated to reflect the change in name and behaviour now, thanks.

Yes ideally we'd have a controller test but unfortunately I just got hit by time constraints.  Ideally we'd merge this test and the ReleaseChecklistServiceTests into an overarching Controller integration test, but I've blown the time estimate that we had for 4909 already and don't wanna push it any further at the present time, if that makes sense?  I could always raise a ticket to test this all at the Controller level, similar to the Publisher function integration test that is currently testing a Service but really would be better to test from the Function level.

Reply:
Cool, no I think that's all fine. The main thing was to sort out the naming and the comments here. Thanks.

Reply:
Cool ta